src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const newDescription = description.replace(/\n/g, ' <br/>');
+
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('afterbegin', `
+    <div class="notification ${type}">
+      <h2 class="title">
+        ${title}
+      </h2>
+      <p>${newDescription}</p>
+    </div>
+  `);
+
+  const divEl = document.querySelector('div');
```

Comment:
```suggestion
  const notification= document.querySelector('div');
```

try to use better and more meangful naming.